Content.Server/Roles/TraitorRole.cs

```diff
@@ -4,7 +4,8 @@
 
 namespace Content.Server.Roles;
 
-public sealed class TraitorRole : AntagonistRole
+[Virtual]
+public class TraitorRole : AntagonistRole
```

Comment:
i get unsealing it but why virtual?

Reply:
to remove warning
making it abstract would mean having to extend it for traitor i think so bad